src/content/docs/it/concepts/islands.mdx

```diff
@@ -0,0 +1,66 @@
+
+<IslandsDiagram>
+  <Fragment slot="headerApp">Intestazione (isola interattiva)</Fragment>
+  <Fragment slot="sidebarApp">Barra laterale (HTML statico)</Fragment>
+  <Fragment slot="main">
+    Contenuto statico come testo, immagini, ecc.
+  </Fragment>
+  <Fragment slot="carouselApp">Carosello di immagini (isola interattiva)</Fragment>
+  <Fragment slot="footer">Piè di pagina (HTML statico)</Fragment>
+  <Fragment slot="source">Fonte: [Architettura delle isole: Jason Miller](https://jasonformat.com/islands-architecture/)</Fragment>
+</IslandsDiagram>
+
+In Astro, è possibile utilizzare qualsiasi framework UI supportato (React, Svelte, Vue, ecc.) per renderizzare le isole nel browser. È possibile combinare diversi framework sulla stessa pagina o semplicemente scegliere il tuo preferito.
+
+La tecnica su cui si basa questo modello architetturale è nota come **idratazione parziale** o **selettiva.** Astro sfrutta questa tecnica dietro le quinte, alimentando automaticamente le tue isole.
+
+## Come funzionano le isole in Astro?
+
+**Astro genera ogni sito web senza alcun JavaScript lato client, per impostazione predefinita.** Utilizza un componente UI frontend costruito con [React](https://react.dev/), [Preact](https://preactjs.com/), [Svelte](https://svelte.dev/), [Vue](https://vuejs.org/), [SolidJS](https://www.solidjs.com/), [AlpineJS](https://alpinejs.dev/) o [Lit](https://lit.dev/), e Astro lo renderizzerà automaticamente in HTML in anticipo per poi rimuovere tutto il JavaScript. Questo mantiene ogni sito veloce per impostazione predefinita, eliminando tutto il JavaScript inutilizzato dalla pagina.
+
+```astro title="src/pages/index.astro"
+---
+// Esempio: Usa un componente React statico sulla pagina, senza JavaScript.
+import MyReactComponent from '../components/MyReactComponent.jsx';
+---
+<!-- 100% HTML, Zero JavaScript caricato sulla pagina! -->
+<MyReactComponent />
+```
+
+Ma talvolta è necessario del JavaScript lato client per creare un'interfaccia utente interattiva. Invece di costringere l'intera pagina a diventare un'applicazione JavaScript simile ad un'applicazione a pagina singola (SPA), Astro ti chiede di creare un'isola.
+
+```astro title="src/pages/index.astro" ins="client:load"
+---
+// Esempio: Usa un componente React dinamico sulla pagina.
+import MyReactComponent from '../components/MyReactComponent.jsx';
+---
+<!-- Questo componente ora è interattivo sulla pagina!
+     Il resto del tuo sito web rimane statico e senza JS. -->
+<MyReactComponent client:load />
+```
+
+Con le Isole Astro, la stragrande maggioranza del tuo sito rimane in HTML e CSS. Nell'esempio sopra, hai appena aggiunto uns singola **isola di interattività**, isolata dal resto della pagina, senza modificare anche il resto della pagina.
```

Comment:
A small typo: `uns` (I suppose it should be `una`)

Reply:
I applied the fixes!